Bullseye/app/src/main/java/com/example/bullseye/MainActivity.kt

```diff
@@ -139,14 +145,21 @@ class MainActivity : AppCompatActivity() {
         return title
     }
     private fun bonusPoints(): Int {
-        val difference = differenceAmount()
-        val  bonus: Int = when {
-        difference == 0 -> 100
-        difference == 1 -> 50
+        val  bonus: Int = when (differenceAmount()) {
+            0 -> 100
+            1 -> 50
             else -> 0
         }
 
 
             return bonus
         }
+// data store attempt, please ignore I don't know much about it but will learn more about it shortly...
```

Comment:
please ignore was messing around with data store to help with orientation but need to do more research